opentelemetry-sdk/tests/metrics/test_metrics.py

```diff
@@ -96,12 +96,31 @@ def test_get_handle(self):
             self.assertEqual(metric.handles.get(label_values), handle)
 
 
-class TestCounterHandle(unittest.TestCase):
-    def test_update(self):
-        handle = metrics.CounterHandle(float, True, False)
-        handle.update(2.0)
-        self.assertEqual(handle.data, 2.0)
+class TestCounter(unittest.TestCase):
+    def test_add(self):
+        metric = metrics.Counter("name", "desc", "unit", int, ("key",))
+        handle = metric.get_handle(("value",))
+        metric.add(("value",), 3)
+        self.assertEqual(handle.data, 3)
+
+
+class TestGauge(unittest.TestCase):
+    def test_set(self):
+        metric = metrics.Gauge("name", "desc", "unit", int, ("key",))
+        handle = metric.get_handle(("value",))
+        metric.set(("value",), 3)
+        self.assertEqual(handle.data, 3)
+
 
+class TestMeasure(unittest.TestCase):
+    def test_record(self):
+        metric = metrics.Measure("name", "desc", "unit", int, ("key",))
+        handle = metric.get_handle(("value",))
+        metric.record(("value",), 3)
```

Comment:
A comment stating that the `record` implementation is missing would be great to avoid confusion.